cdap-watchdog-api/src/main/java/co/cask/cdap/api/metrics/MetricSearchQuery.java

```diff
@@ -24,7 +23,7 @@
 
 /**
  * Defines a query to perform Exploration and Search on {@link MetricStore} data.
- * Given a list of {@link co.cask.cdap.api.dataset.lib.cube.TagValue} this explore query can be used
+ * Given a list of {@link TagValue}s this explore query can be used
  * to find next set of tags available or the measureNames belonging to this tag list.
```

Comment:
to find either the next set of tags available or the measureNames belonging to this tag list.